public/javascripts/ers_csv_scripts.js

```diff
@@ -1,122 +1,96 @@
-	/***********************/
-	/* Check CSV file page */
-	/***********************/
+/***********************/
+/* Check CSV file page */
+/***********************/
 
-	var MAX_CSV_FILESIZE = 100000000// 100 MB
+const MAX_CSV_FILESIZE = 100000000// 100 MB
+document.getElementById("check-file-button").disabled = true;
 
-	function csvFileSizeOK (fileSize) {
-		if (ie<10) {
-			return true
-		} else {
-			if (fileSize > MAX_CSV_FILESIZE) {
-				return false
-			} else {
-				return true
-			}
-		}
-	}
+function csvFileSizeOK(fileSize) {
+    if (ie < 10) {
+        return true
+    } else {
+        return fileSize <= MAX_CSV_FILESIZE;
+    }
+}
 
-	function duplicateFileName (fileName) {
-		var duplicateNameCount = 0
-		$(".input-file-name").each(function(index){
-			if ($(this).val() != "") {
-				if (ie<10) {
-					if ($(this).val().substr($(this).val().lastIndexOf("\\")+1, $(this).val().length) == fileName) {
-						duplicateNameCount++;
-						if (duplicateNameCount > 1) $(this).parent("Div").addClass("fileAlert")
-					}
-				} else {
-					if ($(this)[0].files[0].name == fileName) {
-						duplicateNameCount++;
-						if (duplicateNameCount > 1) $(this).parent("Div").addClass("fileAlert")
-					}
-				}
-			}
-		});
-		if (duplicateNameCount > 1) {
-			return true;
-		} else {
-			return false;
-		}
-	}
+function removeFileAlert() {
+    if (document.querySelector(".govuk-form-group--error")) {
+        document.querySelector(".govuk-form-group--error").classList.remove("govuk-form-group--error")
+    }
+}
 
-	function removeFileAlert () {
-		$(".input-file-name").each(function(index){
-			$(this).parent("Div").removeClass("fileAlert")
-		});
-	}
+function showCSVErrorMsg(e, msg) {
+    const button = document.getElementById("check-file-button");
```

Comment:
If you're to keep `document.getElementById("check-file-button").disabled = true;` in this file then you could define `const button` there instead of having to duplicate this line